.github/workflows/build-test-e2e.yml

```diff
@@ -96,7 +96,7 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --network-timeout 1000000
```

Comment:
Increase network timeout to give dependencies more time to download, as in this [failed e2e job](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/8837987919/job/24269121860)